### ISSUE TITLE
Crashing when parsing befores/afters within a test file

### ISSUE THREAD

**Issue by cthorsen31** [pausan/mocha-list-tests#1]
Hey just found your npm package and was interested in using it for a project I'm currently working on and was curious if you could help with this issue of not reading befores and afters within a test file. 

The issue coming up says it's not defined so I'm assuming there is no functionality to support those keywords yet. 

Fatal Error (try --help for help):
ReferenceError: before is not defined

And it does work when a test file includes only describes and its. 

Not sure if you're actively planning on maintaining this project but I might be able to add functionality for this. Let me know, and thanks!

**Comment by pausan** [comment]
Hi! Sorry for the late answer. Can you provide a simple test example for this? If you also provide the fix and does not break anything, I would be more than happy to merge it.

**Comment by cthorsen31** [comment]
describe('Mocha Timings Test', function() {
      before(function() {
        console.log('this log is done before the test is ran')
      })
      it('should run a test in usingMochaTimings', function() {
        assert.equal(2, 1 + 1);
      })

This will break the functionality because before was never defined when the file was required. Submitted a PR and it works for me for both programmatic use and on CLI. Thanks for the package as well, was a huge help!

**Comment by pausan** [comment]
Hi! Thanks a lot for providing the PR. I've just wrote a similar functionality and had a look at the code just to see we did something very similar in essence. Basically I'm adding all these before/after/... hooks as part of the tree output (but not as part of the tests or suites output). They have an special colon ':' marker at the beginning. This way if somebody wants to use this info, it has it available, and if it does not, it is easy to filter out.

I'm rejecting the PR, but the functionality you were asking for is there.

Thanks again!
